README.developer.md

```diff
@@ -126,15 +126,15 @@ nc <host> <port>
 A node can be run via:
 
 ```bash
-./bin/hirte-agent -n <agent name> -H <host> -P <port>
+./bin/hirte-agent -n <agent name> -H <host>
```

Comment:
Why did you remove the port? This is still a supported cli option as far as I know

Reply:
It is, but the port is set by default, so no need to set it